tools/swift-inspect/Sources/SwiftInspectLinux/AuxVec.swift

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LinuxSystemHeaders
+
+internal class AuxVec {
+  // loads the auxiliary vector for a process
+  public static func load(for process: Process) -> [Int32 : UInt64]? {
+    let filePath = "/proc/\(process.pid)/auxv"
+    guard let fileHandle = FileHandle(forReadingAtPath: filePath) else { return nil }
```

Comment:
Might be good to have a brief comment here about how `Data(contentsOf:)` doesn't work due to the `stat` issue. I can imagine someone trying to "improve" this and being sad.